$NetBSD: patch-Makefile.am,v 1.2 2020/03/12 22:53:12 wiz Exp $

Disable tests, they pull in a lot more dependencies.

Fix unportable test(1) operator.

--- Makefile.am.orig	2018-07-08 22:13:17.000000000 +0000
+++ Makefile.am
@@ -161,9 +161,6 @@ gjs_console_SOURCES = $(gjs_console_srcs
 install-exec-hook:
 	(cd $(DESTDIR)$(bindir) && $(LN_S) -f gjs-console$(EXEEXT) gjs$(EXEEXT))
 
-include Makefile-test.am
-include Makefile-insttest.am
-
 EXTRA_DIST +=					\
 	autogen.sh				\
 	COPYING.LGPL				\
@@ -216,7 +213,7 @@ prepare-minor-release: $(DISTNAME) prepa
 
 upload-release: $(DISTNAME) Makefile
 	git log origin/master..master
-	@echo -n "Ok to push? [y/N] "; read ans; test x$$ans == xy || exit 1
+	@echo -n "Ok to push? [y/N] "; read ans; test x$$ans = xy || exit 1
 	git push --tags origin master:master
 	scp $(DISTNAME) master.gnome.org:
 	ssh master.gnome.org install-module $(DISTNAME)
